include/cxx.h

```diff
@@ -235,15 +235,27 @@ class Vec final {
 
   class const_iterator {
   public:
+    using difference_type = ptrdiff_t;
     using value_type = typename std::add_const<T>::type;
+    using pointer = typename std::add_pointer<
+        typename std::add_const<T>::type>::type;
     using reference = typename std::add_lvalue_reference<
         typename std::add_const<T>::type>::type;
+    using iterator_category = std::forward_iterator_tag;
 
+    const_iterator(): pos(nullptr), stride(0) { }
```

Comment:
We don't need to initialize the members because the default constructed iterator isn't required to have any particular behavior.

After deleting the other constructor, this class gets an implicitly declared default constructor which is sufficient for the purpose of forward_iterator, so this can be removed.

Reply:
Makes sense.  Removed in 72ceca3.